Dockerfile

```diff
@@ -9,7 +9,7 @@ VOLUME ["/var/lib/odoo"]
 EXPOSE 8069 8072
 
 # Subimage triggers
-ONBUILD ENTRYPOINT ["/opt/odoo/common/entrypoint.sh"]
+ONBUILD ENTRYPOINT ["/opt/odoo/common/entrypoint"]
```

Comment:
Hmmmm why add a file extension? I used no extension here just in case we decide to switch the scripting language again for some reason. 

I don't feel like the filename of the binary should be intrinsically linked to the code type - the extension is a humanization factor that IMO is not really relevant when the pieces are managed with automation.